.github/workflows/release.yml

```diff
@@ -6,7 +6,7 @@ on:
       - "v*"
 
 jobs:
-  aarch64-apple-darwin:
+  aarch64-apple-darwin-release:
```

Comment:
I renamed these jobs to avoid name conflicts with the jobs defined in `build.yml`.